target/linux/bcm27xx/patches-5.15/950-0706-media-i2c-imx219-Scale-the-pixel-clock-rate-for-the-.patch

```diff
@@ -15,44 +15,44 @@ Signed-off-by: Naushir Patuck <naush@raspberrypi.com>
 
 --- a/drivers/media/i2c/imx219.c
 +++ b/drivers/media/i2c/imx219.c
-@@ -153,6 +153,9 @@ struct imx219_mode {
- 
+@@ -160,6 +160,9 @@ struct imx219_mode {
  	/* Default register values */
  	struct imx219_reg_list reg_list;
-+
+ 
 +	/* Relative pixel clock rate factor for the mode. */
 +	unsigned int rate_factor;
```

Comment:
Please add the new attribute to the end of the struct.

Reply:
Done, will push shortly.